devito/dle/backends/advanced.py

```diff
@@ -196,82 +196,25 @@ def _simdize(self, nodes, state):
                 except KeyError:
                     aligned = []
                 if aligned:
-                    simd = omplang['simd-for-aligned']
+                    simd = Ompizer.lang['simd-for-aligned']
                     simd = as_tuple(simd(','.join([j.name for j in aligned]),
                                     simdinfo[get_simd_flag()]))
                 else:
-                    simd = as_tuple(omplang['simd-for'])
+                    simd = as_tuple(Ompizer.lang['simd-for'])
                 mapper[i] = i._rebuild(pragmas=i.pragmas + ignore_deps + simd)
 
         processed = Transformer(mapper).visit(nodes)
 
         return processed, {}
 
     @dle_pass
-    def _ompize(self, nodes, state):
+    def _parallelize(self, iet, state):
```

Comment:
note for reviewers: nothing really changed below, I've just moved (and split over multiple methods) this code to ``parallelizer.py``. This gives YASK (and future backends) the possibility of specializing parallelization rules and strategies